Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/VehicleRegistrationCertificate.hs

```diff
@@ -229,9 +251,94 @@ onVerifyRC verificationReq output = do
             associatedOn = now,
             associatedTill = convertTextToUTC (Just "2099-12-12"),
             consent = True,
-            consentTimestamp = now
+            consentTimestamp = now,
+            isRcActive = False
           }
 
+makeRcActiveOrInactive :: (Id Person.Person, Id DM.Merchant) -> RcActiveInactiveReq -> Flow APISuccess
+makeRcActiveOrInactive (driverId, merchantId) req@RcActiveInactiveReq {..} = do
+  if isActivate
+    then do
+      validateCanRcActive (driverId, merchantId) rcNo
```

Comment:
Why value returned by this function is being ignored.

Reply:
I have changed the returned type that's why.